src/views/Header.js

```diff
@@ -218,16 +223,33 @@ class Header extends React.Component<void, HeaderProps, void> {
       subView = defaultRenderer(subViewProps);
     }
 
-    if (subView === null) {
-      return null;
-    }
-
     const pointerEvents = offset !== 0 || isStale ? 'none' : 'box-none';
+
+    // Only measure `title` component
+    const onLayoutIOS = Platform.OS === 'ios' && name === 'title'
+      ? (e: LayoutEvent) => {
+        this.setState({
+          widths: {
+            ...this.state.widths,
+            [index]: e.nativeEvent.layout.width,
+          },
+        });
+      }
+      : undefined;
+
+    const width = name === 'left' || name === 'right'
+      ? this.state.widths[index]
+      : undefined;
+
     return (
       <Animated.View
         pointerEvents={pointerEvents}
+        onLayout={onLayoutIOS}
         key={`${name}_${key}`}
         style={[
+          width && {
+            width: (props.layout.initWidth - width) / 2,
```

Comment:
I think the `width` variable needs a better name. it's not obvious what this line does